maps/utk.yml

```diff
@@ -5955,6 +5956,38 @@ properties:
     requirement: optional
     sample_values:
     - http://id.loc.gov/authorities/names/n2017180154
+  work_type:
+    available_on:
+      class:
+      - GenericWork
+      - Image
+      - Video
+      - Audio
+      - Pdf
+      - Book
+      - CompoundObject
+      - Newspaper
+    cardinality:
+      minimum: 1
+      maximum: 1
+    multi_value: true
+    controlled_values:
+      format: http://www.w3.org/2001/XMLSchema#anyURI
+      sources:
+      - utk
+    definition:
+      default: Describes the local work type of the work.
+    display_label:
+      default: Work Type
+    index_documentation: displayable, searchable
```

Comment:
I don't think "index_documentation" actually does anything, but I don't think we want it to be able to be displayed.